### DISCUSSION TITLE
Update sites (also nightly)

### DISCUSSION THREAD

**Discussion by iloveeclipse** [eclipse-oomph/oomph#54] [General]
I'm managing in-house Oomph based installation.

For that one we don't want to refer to any "external" update site on the Internet, simply because not all sites are connected to internet with adequate speed or not connected at all (many urls are just blocked).

So we have our own update sites for Oomph installer and time to time I mirror the p2 sites on a platform update or just a nightly build if I need some "intermediate" build.

There are at least 3 pieces involved:
- Installer binary build
- Installer update site
- Oomph update site

In the past (for 4.25 SDK release), I've seen / used integration build taken from https://ci.eclipse.org/oomph/job/integration/ but that seem to be changed. Where are the update sites / binary build artifacts (zipped or online) for Oomph now? I'm especially interested in 4.30 RC2.

Unfortunately I couldn't find any link on https://projects.eclipse.org/projects/tools.oomph/downloads

**Comment by iloveeclipse** [comment]
OK, I've found **Oomph** update site: https://download.eclipse.org/oomph/updates/
Missing: Installer binary build drops & Installer update site

**Comment by iloveeclipse** [comment]
And I believe I've found also installer binary drops: https://download.eclipse.org/oomph/epp/2023-12/

Last piece missing: **Installer update site**?

**Comment by merks** [comment]
Sorry, the  documentation on the ci instance was out-of date.  It now references the correct job.

The downloads on the portal is atrocious as well.

The wiki provides better information, but all needs to be migrated:

- https://wiki.eclipse.org/Oomph
- https://wiki.eclipse.org/Oomph#Update_Sites
- https://wiki.eclipse.org/Eclipse_Installer

The product update sites are not maintained long term but rather updated regularly:

- https://download.eclipse.org/oomph/products/repository/
- https://download.eclipse.org/oomph/products/latest/repository/

The former is updated for release or milestone build, so currently contains the release.  The later is changed for each build including for nightly builds.  The parent folder of each contains the corresponding binaries.  And yes, you've found the epp binaries.  There are the restricted ones that in simple mode will only install that version of the products from the catalog.

Clearly we could use some improvements in documentation!

**Comment by merks** [comment]
So for your use case, look here for Oomph releases as installed into IDEs:

https://download.eclipse.org/oomph/drops/release/

Look here for installer releases:

https://download.eclipse.org/oomph/products/repository/

But get them while they last, because these will be replaced for M1.

**Reply by iloveeclipse** [reply]
Many thanks Ed!

**Reply by merks** [reply]
You're very welcome.  It's good you ask too, it reminds me just how much work there is needed to document things properly going forward.
